config/routes.rb

```diff
@@ -8,4 +9,18 @@
       resources :likes, only: %i[new create]
     end
   end
+
+  resources :users, only: [] do
+    resources :posts, only: %i[index] do
+      resources :comments, only: %i[index create]
+    end
+  end
+
+  namespace :api, default: {format: :json} do
+    namespace :v1 do
+      resources :posts, only: %i[index] do
+        resources :comments, only: %i[index create]
+      end
+    end
```

Comment:
Hello there! I noticed that although you have made some updates to your code after the previous review, it still isn't working properly. You added some route configurations between lines 13 to 17 as requested by the previous reviewer, but unfortunately, these routes have not been configured within the API scope. The API part of your code is actually between lines 19 to 24. The previous reviewer also mentioned that there should be a user route within the `v1` routes, followed by posts and comments, for example, `/api/v1/users/:user_id/posts...`. I kindly request you to update your code to ensure that it works as requested.

```suggestion
  namespace :api, default: {format: :json} do
    namespace :v1 do
      resources :users, only: [] do
        resources :posts, only: %i[index] do
          resources :comments, only: %i[index create]
        end
      end
    end
  end 
```

Reply:
please if there is anny issues after this I need practical code to learn from it please I have tried and do many research but that's all I could find